velox/type/Conversions.h

```diff
@@ -385,15 +386,37 @@ struct Converter<
   }
 
   static T cast(folly::StringPiece v) {
-    return cast<folly::StringPiece>(v);
+    T output;
```

Comment:
This code repeats 3 times. Would it be possible to reduce copy-paste?

Reply:
Hmm, let me refactor it.